packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -143,6 +144,7 @@ const emit = defineEmits<{
 
   &:disabled {
     cursor: default;
+    pointer-events: none;
```

Comment:
We must avoid `pointer-events: none;` as it introduces other side effects.
Instead, you could only apply the `:hover` when the button is also `:enabled`